src/LuaLib.ts

```diff
@@ -46,8 +48,10 @@ export enum LuaLibFeature {
     Iterator = "Iterator",
     LuaIteratorSpread = "LuaIteratorSpread",
     Map = "Map",
+    Match = "Match",
     MathAtan2 = "MathAtan2",
     MathSign = "MathSign",
+    Modulo = "Modulo",
```

Comment:
If this modulo is only required for 50, maybe we should call this `Modulo50`

Reply:
If we still need it (see above), I would agree with calling it `Modulo50`.

Reply:
Renamed with https://github.com/TypeScriptToLua/TypeScriptToLua/pull/1263/commits/236bd774ece164145eeb334143f2ade012e91ad4.